Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN pushd /coreapi && \
     # needed for DB migrations
     find coreapi/ -mindepth 1 -maxdepth 1 \( ! -name 'alembic*' -a ! -name hack \) -exec rm -rf {} +
 
-RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
+RUN pip3 install git+https://github.com/yzainee/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
```

Comment:
@yzainee install has to point to the `fabric8-analytics` repo.

Reply:
@yzainee please verify the worker commit hash in the Dockerfile and Dockerfile.rhel as well

Reply:
Typo.. rectified